Assignment6/createClient.js

```diff
@@ -0,0 +1,16 @@
+exports.createClient = (app, request, url, headers) => {
+  app.post('/clients', (req, res) => {
+    request.post( {
+   		url : url + ".json",
+   		headers : headers,
+   		body : JSON.stringify(req.body)
+   		},
+   		(error, response, body) => {
+   		  if (error)
+   		    console.log(error);
```

Comment:
you're just logging error. Also return error object in response in case of error